tensorflow/tools/dist_test/python/mnist_replica.py

```diff
@@ -206,14 +206,16 @@ def main(unused_argv):
           init_op=init_op,
           local_init_op=local_init_op,
           ready_for_local_init_op=ready_for_local_init_op,
-          recovery_wait_secs=1,
+          save_model_secs=60,
```

Comment:
This code here is for testing gRPC (distributed) sessions only. Can you explain why you need this change?